src/common/config/constants.ts

```diff
@@ -216,6 +216,10 @@ export const MS_PER_DAY = 1000 * 60 * 60 * 24;
 
 export const NO_SUBNET_SET = 'NO_SUBNET_SET';
 
+export const NETWORK_TYPE_OVN = 'OVNKubernetes';
+export const NETWORK_TYPE_SDN = 'OpenShiftSDN';
```

Comment:
What is the difference between this and using strings in the code?

Reply:
It's generally a better practice to reference values via variables.
This avoids:
- Typing errors
- Having to replace the code in many different place if the values change in the future.  
- It also lets you give "better" names. If the name is confusing, etc, your constant can avoid that confusion.

With the constants, if the values in the Backend change, we would only need to replace it here.

Reply:
Also, the bundler cannot minify plain strings... Think about how many times the string "OVNKubernetes" (or the other one) would appear in the final bundle.